docs/filters.md

```diff
@@ -148,7 +148,7 @@ Matches all issues with a body (or description) containing the given string.
 
 *Expects a string*
 
-Matches all issues associated with any milestones whose names contain the given string.
+Matches all issues associated with any milestones whose names contain the given string. `current` or `curr` can be used to refer to an open milestone with earliest due date, or an overdue open milestone with open issues if any. `prev`, `prev-1`, `prev-2`, `next`, `next+1, `next+2`, etc. can be used to refer to milestones before or after `current`, sorted by due date.
 
```

Comment:
Hmm, it depends on if we want users to the think of the contents of a qualifier as a actual arithmetic expression rather than just a string literal, i.e. if we may have arithmetic operations in future. Otherwise I think just 'current' is fine, since there's less syntax to remember.